interview-questions.md

```diff
@@ -8,36 +8,82 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Add the update information through migrate.
 
 Researched answer:
 
+1. Go to terminal
+
+2. Generate a new migration
+
+3. Open the migration file and locate the change method
+
+4. Save the file
+
+5. Migrate through the command "rails db:migrate
+
+foreign key name: cohort_id
+
+The cohort model.
```

Comment:
The foreign key should be in the Student model. I would elaborate some more on what you have to do in the change method.